app/sections/promotion-grid/item.tsx

```diff
@@ -0,0 +1,216 @@
+import type {
+  HydrogenComponentProps,
+  HydrogenComponentSchema,
+} from '@weaverse/hydrogen';
+import { forwardRef } from 'react';
+import { Image } from '@shopify/hydrogen';
+import { CSSProperties } from 'react';
+
+interface PromotionItemProps extends HydrogenComponentProps {
+  backgroundImage: {
```

Comment:
Import `WeaverseImage` type from `@weaverse/hydrogen` instead of defining type for `backgroundImage`